packages/webcomponents/src/components/business-forms/payment-provisioning/terms-and-conditions/business-terms-conditions-text.tsx

```diff
@@ -9,21 +9,21 @@ export class BusinessTermsConditionsText {
   render() {
     return (
       <div class='terms-container'>
-        <div class="card terms-conditions-card">
-          <div class="card-body terms-card-body">
-            <h2 class="card-title">JustiFi Technologies Inc. Terms and Conditions</h2>
+        <div class="card terms-conditions-card" part="card">
+          <div class="card-body terms-card-body" part="card-body">
+            <h2 class="card-title" part="card-title">JustiFi Technologies Inc. Terms and Conditions</h2>
 
-            <p class='card-text'>The Payment Terms and Conditions described herein constitute a legal agreement (“Agreement” or “JustiFi Technologies Inc. Terms”) between your business (the “Merchant”, “you” or “your”) and JustiFi Technologies Inc., organized under the laws of Minnesota with offices at 555 Vandalia St. St Paul, MN 55114, Suite #105. (“JustiFi”, “we”, “us”, or “our”).</p>
+            <p class='card-text' part="card-text">The Payment Terms and Conditions described herein constitute a legal agreement (“Agreement” or “JustiFi Technologies Inc. Terms”) between your business (the “Merchant”, “you” or “your”) and JustiFi Technologies Inc., organized under the laws of Minnesota with offices at 555 Vandalia St. St Paul, MN 55114, Suite #105. (“JustiFi”, “we”, “us”, or “our”).</p>
 
             <br />
-            <h3 class="card-subtitle mb-2 text-muted">A. The JustiFi Payments Service</h3>
+            <h3 class="card-subtitle mb-2 text-muted" part="card-subtitle">A. The JustiFi Payments Service</h3>
             <br />
 
-            <h4 class="card-subtitle">Our Role</h4>
-            <p class='card-text'>The JustiFi Payments service (“JustiFi Payments” or the “Payments Services”) is a: (i) payment account boarding; (ii) payment underwriting; and, (iii) payment data transmission service that helps you integrate with a payment processor (the “Processor”), as described more fully in this Agreement.</p>
-            <p class='card-text'>JustiFi is not a bank, payment institution, or money services business, but is instead, respectively, a supplier of the JustiFi Platform Services supplied under the JustiFi Platform Terms and of the Payments Services under this Agreement.</p>
+            <h4 class="card-subtitle" part="part-subtitle">Our Role</h4>
```

Comment:
small typo - I think this is supposed to be "card-subtitle" instead of "part-subtitle"